lib/puppet/transaction/report.rb

```diff
@@ -183,7 +194,7 @@ def initialize(kind, configuration_version=nil, environment=nil, transaction_uui
     @host = Puppet[:node_name_value]
     @time = Time.now
     @kind = kind
-    @report_format = 5
+    @report_format = 6
```

Comment:
Are we really bumping the version for every new field? Is that our versioning policy here @kylog? Seems like we've broken this a few times in the past if thats true.